http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/config/KubernetesAppConfig.scala

```diff
@@ -33,6 +33,9 @@ sealed trait KubernetesAppConfig extends Product with Serializable {
   // corresponds to a specific app type and cloud provider.
   def cloudProvider: CloudProvider
   def appType: AppType
+
+  // Only CROMWELL, ALLOWED appTypes have replicas defined properly and used for pause/resume.
+  def numOfReplicas: Option[Int]
```

Comment:
Based on what you've said about how this works I assume `replicaCount` will always be either 0 or 1 right now. The app won't Just Work with replica count > 1, might be good to have some validation somewhere checking that.

Reply:
yea..so right now, I've defaulted cromwell replica to 1....and assume that's the number it needs to scale up to...down the line, I think we should parameterize that in the cromwell chart, and let Leo pass that in so that there won't be mismatch